ChargeApp/app/models/charge.rb

```diff
@@ -0,0 +1,3 @@
+class Charge < ActiveRecord::Base
+  belongs_to :customer
+end
```

Comment:
There is an interesting gem in common use: 'monetize' that makes working with money really nice. A supporting gem: 'money-rails' allows us to use those features on ActiveRecord model fields.